sql/expression/function/uuid.go

```diff
@@ -25,6 +27,13 @@ import (
 	"github.com/dolthub/go-mysql-server/sql/types"
 )
 
+// Global state for UUID_SHORT function
+var (
+	uuidShortMu      sync.Mutex
+	uuidShortCounter uint64
+	uuidShortStartup = uint64(time.Now().Unix())
```

Comment:
I think this fine for now since we don't save the server start up time anywhere, but leave a TODO to use that system variable whenever we implement it.